packages/material-ui/src/styles/createPalette.d.ts

```diff
@@ -85,6 +88,4 @@ export interface PaletteOptions {
   getContrastText?: (background: string) => string;
 }
 
-//export type PaletteOptions = DeepPartial<Palette>;
```

Comment:
Why?

Reply:
This was commented out. Why do we want to preserve the commented out code?

Reply:
I was wondering why it was commented out; Let's remove it then :)